client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx

```diff
@@ -7,11 +7,36 @@ import profCover from '../../../Images/profcover.jpg';
 import profimg from '../../../Images/profimg.jpg';
 import doggo from '../../../Images/dog1.jpg';
 import doggo1 from '../../../Images/dog2.jpg';
+import loading from '../../../Images/loading.gif';
+import { useState, useEffect } from "react";
+import  getProfileDetails  from '../../../helpers/APICalls/profile';
+import { useAuth } from "../../../context/useAuthContext";
 
 const ProfileDetails = (): JSX.Element => {
 
     const classes = useStyles();
 
+    const { loggedInUser } = useAuth();
+
+    const [ data, setData ] = useState<any>();
```

Comment:
Maybe use the Profile interface (if we have one) as the type of the  state here?